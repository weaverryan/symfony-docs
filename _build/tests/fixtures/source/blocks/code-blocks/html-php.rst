
.. code-block:: html+php
    <!-- views/layout.php -->
    <!doctype html>
    <html>
        <head>
            <title>
                <?php echo 'title'; ?>
            </title>
        </head>
        <body>
            <?php echo 'body'; ?>
        </body>
    </html>
