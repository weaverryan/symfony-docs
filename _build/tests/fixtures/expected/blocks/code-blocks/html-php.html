<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="literal-block notranslate">
    <div class="highlight-html+php">
        <table class="highlighttable">
            <tr>
                <td class="linenos">
                    <div class="linenodiv">
                        <pre> 1 2 3 4 5 6 7 8 9
10
11
12</pre>
                    </div>
                </td>
                <td class="code">
                    <div class="highlight">
                        <pre class="hljs html">
                            <span class="hljs-comment">&lt;!-- views/layout.php --&gt;</span>
<span class="hljs-meta">&lt;!doctype html&gt;</span>
<span class="hljs-tag">&lt;<span class="hljs-name">html</span>&gt;</span>
                            <span class="hljs-tag">&lt;<span class="hljs-name">head</span>&gt;</span>
                            <span class="hljs-tag">&lt;<span class="hljs-name">title</span>&gt;</span>
                            <span class="php"><span class="hljs-meta">&lt;?php</span> <span class="hljs-keyword">echo</span> <span class="hljs-string">'title'</span>; <span class="hljs-meta">?&gt;</span></span>
                            <span class="hljs-tag">&lt;/<span class="hljs-name">title</span>&gt;</span>
                            <span class="hljs-tag">&lt;/<span class="hljs-name">head</span>&gt;</span>
                            <span class="hljs-tag">&lt;<span class="hljs-name">body</span>&gt;</span>
                            <span class="php"><span class="hljs-meta">&lt;?php</span> <span class="hljs-keyword">echo</span> <span class="hljs-string">'body'</span>; <span class="hljs-meta">?&gt;</span></span>
                            <span class="hljs-tag">&lt;/<span class="hljs-name">body</span>&gt;</span>
                            <span class="hljs-tag">&lt;/<span class="hljs-name">html</span>&gt;</span>
                        </pre>
                    </div>
                </td>
            </tr>
        </table>
    </div>
</div>
</body>
</html>
